### PR TITLE
Added comments to each step in startService.sh

### PR DESCRIPTION
**Pull request by jshaptala**
Added comments to each step in startService.sh; @maxim-voronko please review and merge if OK.

### CODE REVIEW

File: files/startService.sh [hunk 1]
```diff
@@ -1,8 +1,15 @@
 #!/bin/sh
+# obtain the name of the directory containing a script
 cd `dirname $0`
```
Comment (maxim-voronko) [inline files/startService.sh]: Not obtain, change directory.

File: files/startService.sh [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+# assign the path for a variable where AppServer is located
 appserver_path=`pwd`
```
Comment (maxim-voronko) [inline files/startService.sh]: assign the current path to variable

File: files/startService.sh [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+# export the libraries required for the service start
 export LD_LIBRARY_PATH=${appserver_path}/lib
+# run script to configure samples and shared dictionaries using the host name
 perl configureFiles.pl $2
+# activate a license automatically
 ./AppServerX -activateLicense $1 -y
+#service apache2 start #start Apache HTTP Server
 service apache2 start
```
Comment (maxim-voronko) [inline files/startService.sh]: We have some two comments here. We need only one.

File: files/startService.sh (hunk 1 above)
Comment (jshaptala) [reply files/startService.sh]: @maxim-voronko updated.

File: files/startService.sh (hunk 2 above)
Comment (jshaptala) [reply files/startService.sh]: Updated.

File: files/startService.sh (hunk 3 above)
Comment (jshaptala) [reply files/startService.sh]: Updated.

File: files/startService.sh (hunk 1 above)
Comment (maxim-voronko) [reply files/startService.sh]: Сhange the working directory to the directory that contains the script.

File: files/startService.sh (hunk 2 above)
Comment (maxim-voronko) [reply files/startService.sh]: I think better use just: 'Assign the current path to variable.'  -- and that is all.
 a variable where AppServer is located -- it is very strange.  ( Переменная где Сервер расположен.)

**Pull request merged**